cdk-infra/lib/constructs/api/webhook-env-construct.ts

```diff
@@ -19,15 +19,16 @@ export class WebhookEnvConstruct extends Construct {
     const dbName = StringParameter.valueFromLookup(this, `${ssmPrefix}/atlas/dbname`);
     const dbUsername = StringParameter.valueFromLookup(this, `${ssmPrefix}/atlas/username`);
     const dbHost = StringParameter.valueFromLookup(this, `${ssmPrefix}/atlas/host`);
+    const jobCollection = StringParameter.valueFromLookup(this, `${ssmPrefix}/atlas/collections/job/queue`);
 
-    const dbPassword = secureStrings[`${ssmPrefix}/atlas/password`];
+    const dbPassword = secureStrings['MONGO_ATLAS_PASSWORD'];
     this.environment = {
       ...secureStrings,
```

Comment:
we have a few places where [env vars are set via Dockerfile.](https://github.com/mongodb/docs-worker-pool/blob/meta/makefiles/shared.mk#L58-L66) might be a good place to start centralizing env files

Reply:
I think this is a good idea, but I am worried about how this might impact the current infrastructure if we stop setting environment variables in the makefiles. We might need to think about how we can eventually migrate all env vars over to when we define them for the infra.